app/models/katello/alternate_content_source.rb

```diff
@@ -110,8 +116,35 @@ def audit_updated_products(old_product_ids)
       write_audit(action: 'update', comment: _('Products updated.'), audited_changes: { 'product_ids' => [old_product_ids, product_ids] })
     end
 
-    def self.humanize_class_name
+    def self.humanize_class_name(_name = nil)
       "Alternate Content Sources"
     end
+
+    # Disallow static properties from being modified on update
+    def constraint_acs_update
+      if changes.keys.include? "content_type"
+        errors.add(:content_type, "cannot be modified once an ACS is created")
+      end
+      if changes.keys.include? "alternate_content_source_type"
+        errors.add(:alternate_content_source_type, "cannot be modified once an ACS is created")
+      end
+    end
+
+    # Disallow --ssl-* properties from being set for simplified ACS
+    # Must be done in method as workaround for validation printing the incorrect
+    # field string.
```

Comment:
I don't really consider this a workaround :smile:  It's just another option Rails offers for writing validations. I'd remove this comment